File: web-src/sampleRobot.raw.js
function main(input) {
  let actions = {}
  console.log(input)
  console.log(displayMap(input.state.objs, input.state.map))

  for (let id of input.state.teams[input.team]) {
    actions[id] = {
      type_: input.state.turn % 2 == 0 ? 'Move' : 'Attack',
      direction: input.team == 'red' ? 'Right' : 'Down',
    }
  }

  return { actions }
}

File: frontend/src/sampleRobot.raw.py
def main(inp):
  actions = {}
  for id in inp['state']['teams'][inp['team']]:
    actions[id] = {
      'direction': 'Down'
    }
    if inp['state']['turn'] % 2 == 0:
      actions[id]['type_'] = 'Move'
      actions[id]['direction'] = 'Right'
    else:
      actions[id]['type_'] = 'Attack'
      actions[id]['direction'] = 'Down'

  return { 'actions': actions }
